public/app/core/components/NavBar/utils.ts

```diff
@@ -79,7 +80,11 @@ export const enrichConfigItems = (items: NavModelItem[], location: Location<unkn
 };
 
 export let getSupportBundleFooterLinks = (cfg = config): FooterLink[] => {
-  if (!cfg.featureToggles.supportBundles) {
+  const hasAccess =
+    contextSrv.hasAccess(AccessControlAction.ActionSupportBundlesCreate, contextSrv.isGrafanaAdmin) ||
+    contextSrv.hasAccess(AccessControlAction.ActionSupportBundlesRead, contextSrv.isGrafanaAdmin);
+
+  if (!cfg.supportBundlesEnabled || !hasAccess) {
```

Comment:
nit: here and in `SupportBundles.tsx` we require server admin permissions to display support bundle components if RBAC is disabled. I wonder if there's an easy way to take into account the `ReqGrafanaAdmin` setting on frontend.

If it's too complex, I think we can ignore it for now (or list it in known limitations?), as the option to disable RBAC is going away soon anyway.

Reply:
yeah, I saw that but since this is a new feature I thought there's no expectation for it to work without RBAC so I left it as is. 

Adding it to the documentation as a known limitation is nice but I thought disabling RBAC was not something supported already and as such it's expected behavior?

Reply:
I think disabling RBAC is in the grey zone of being strongly discouraged, but still supported (and I'm fairly sure that some customers have it disabled). However, I think it's fine to assume that support bundles will be used with RBAC (as this is a new feature and that is still in beta, and as the option to disable RBAC won't be applicable from 10 onwards).